src/runtime/aot_executor/aot_executor.cc

```diff
@@ -107,7 +108,7 @@ PackedFunc AotExecutor::GetFunction(const std::string& name,
   } else if (name == "set_input_zero_copy") {
     return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
       if (String::CanConvertFrom(args[0])) {
-        int in_idx = this->GetInputIndex(args[0].operator String());
+        int in_idx = this->GetInputIndex(tvm::runtime::SanitizeName(args[0].operator String()));
```

Comment:
Does it also make sense to add a python test that passes in names like `input:0` to test that this functionality doesn't regress for the various AoT commands?

Reply:
yeah it makes sense. I will add a test.

Reply:
I added a test which exams both mangled name and the original name